kernel/packages/decentraland-amd/tsconfig.json

```diff
@@ -1,10 +1,10 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "target": "es2017",
```

Comment:
```suggestion
    "target": "es5",
```

Reply:
It uses sets and array.from, there is no real reason to stick to ES5 anymore

Reply:
Thanks! That's related to the `lib`, right? Let's leave the `target` as es5